packages/sdk/src/latest-block/latest-block.js

```diff
@@ -21,5 +21,17 @@ export function latestBlock(...args) {
     )
   }
 
+  console.warn(
```

Comment:
We should add an entry for this warning in WARNINGS.MD with a prescribed migration path